package.json

```diff
@@ -75,5 +75,8 @@
   "engines": {
     "node": ">=16.0.0",
     "npm": ">=8.0.0"
+  },
+  "dependencies": {
+    "@primer/primitives": "^9.1.1"
```

Comment:
Can we remove this? It's a duplicate of the one below

Reply:
🤦